openstf/model/train.py

```diff
@@ -144,14 +204,37 @@ def train_model_for_specific_pj(
     train_data, validation_data, test_data = split_data_train_validation_test(
         clean_data_with_features
     )
+    return {
+        "train_data": train_data,
+        "validation_data": validation_data,
+        "test_data": test_data,
+    }
 
-    context.perf_meter.checkpoint("preprocessing")
 
+def train_for_specific_pj(model_trainer, split_input_data):
+    """Train model.
+
+    Args:
+        model_trainer (MLModelType): model trainer
+        split_input_data (dict): dict of cleaned data with features, split in train,
+            validation and test
+    """
     # Train model
-    model_trainer.train(train_data, validation_data)
+    model_trainer.train(split_input_data.train_data, split_input_data.validation_data)
```

Comment:
You can't index a dictionary like this right? This should fail a unit test?

Reply:
It did and it has been changed. odd that it is still here, I pushed that change long time ago.